pkg/wrapper/nmt_wrapper_test.go

```diff
@@ -26,7 +26,7 @@ func TestPushErasuredNamespacedMerkleTree(t *testing.T) {
 		tree := NewErasuredNamespacedMerkleTree(uint64(tc.squareSize), 0)
 
 		// push test data to the tree
-		for _, d := range generateErasuredData(t, tc.squareSize, appconsts.DefaultCodec()) {
+		for _, d := range generateErasuredData(t, tc.squareSize, rsmt2d.NewLeoRSCodec()) {
```

Comment:
will this change also need to be tested with QGB messages?

Reply:
it shouldn't, but this still reminded me to switch that back to the default https://github.com/celestiaorg/celestia-app/pull/987/commits/97feb2ca4faec88b7a37827d20bc6db9523395f0

Reply:
it shouldn't because this only has an effect of the erasure data.